### PR TITLE
slight restructuring of template

### PR DESCRIPTION
**Pull request by kwsutter**
I modeled this after what we did with MicroProfile.  It breaks apart the front matter from the license from the actual content.  I also introduced some of the naming convention that we use so that you know what version of the pdf or html you are dealing with. Hope it helps!

### CODE REVIEW

File: src/main/asciidoc/license-epl2.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
+[subs="normal"]
```
Comment (bshannon) [inline src/main/asciidoc/license-epl2.adoc]: This is the Eclipse Foundation Specification License so the file name should be license-efsl.adoc.

File: src/main/asciidoc/wombat-spec.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// Copyright (c) 2017-2019 Contributors to the Eclipse Foundation
+//
+
+= Jakarta Wombat
+:authors: Jakarta Wombat Team, https://projects.eclipse.org/projects/ee4j.wombat
+:email: https://dev.eclipse.org/mailman/listinfo/wombat-dev
+:version-label!:
+:sectanchors:
+:doctype: book
+:license: Eclipse Public License v2.0
```
Comment (bshannon) [inline src/main/asciidoc/wombat-spec.adoc]: Eclipse Foundation Specification License v1.0

File: src/main/asciidoc/wombat-spec.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+:title-logo-image: image:wombat-1.png[pdfwidth=4.25in,align=right]
+endif::[]
+
+// == License
+
+include::license-epl2.adoc[]
+
+// == Wombat
+
+include::wombat.adoc[]
```
Comment (bshannon) [inline src/main/asciidoc/wombat-spec.adoc:28]: This should be Wombat.adoc, or the file should be renamed to wombat.adoc.

**Review by bshannon — CHANGES_REQUESTED** [review]
Generally I like the improvements, but there's a few problems...

Something here changed the html version so that it no longer includes the TOC on the left.
I don't know if that was on purpose or not, but I sort of liked it.

Also, the formatting of the license text isn't correct, the headings are being ignored.

File: src/main/asciidoc/wombat-spec.adoc (hunk 3 above)
Comment (kwsutter) [reply src/main/asciidoc/wombat-spec.adoc:28]: Will change to wombat.adoc in git.

File: src/main/asciidoc/wombat-spec.adoc (hunk 2 above)
Comment (kwsutter) [reply src/main/asciidoc/wombat-spec.adoc]: Yep, my mistake.

File: src/main/asciidoc/license-epl2.adoc (hunk 1 above)
Comment (kwsutter) [reply src/main/asciidoc/license-epl2.adoc]: Yep, my mistake.  Will correct.

**Comment by kwsutter** [comment]
Thanks for the review.  Not sure about the TOC change in html.  I was focused on the pdf generation.  I'll experiment and see what I (accidentally) changed.  I put the license in a literal block to highlight it.  That might have removed the formatting.  I'll do a bit more experimenting and re-commit.

**Comment by kwsutter** [comment]
Looks like the `<backend>` type affected the TOC generation.  If I change it to html5, I can generate the TOC.  The generated doc looks good to me.  Was there something special where you required xhtml?  I also changed the formatting of the license page to only highlight the front matter, but the license itself is formatted with headings.  I'll push shortly..

File: src/main/asciidoc/wombat-spec.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+:email: https://dev.eclipse.org/mailman/listinfo/wombat-dev
+:version-label!:
+:sectanchors:
+:doctype: book
+:license: Eclipse Foundation Specification License v1.0
+:source-highlighter: coderay
+:toc: left
+:toclevels: 4
+:sectnumlevels: 4
+ifdef::backend-pdf[]
```
Comment (bshannon) [inline src/main/asciidoc/wombat-spec.adoc:16]: Something in here is turning on :sectnums: before the license (which shouldn't have section numbers), but only for the pdf version.

**Review by bshannon — CHANGES_REQUESTED** [review]
Looks great!  One minor issue marked below.

**Comment by bshannon** [comment]
Oh, and is the image being added to the cover page of the pdf on purpose?
It looks fine in this example, but I don't see how to control it.

File: src/main/asciidoc/wombat-spec.adoc (hunk 4 above)
Comment (kwsutter) [reply src/main/asciidoc/wombat-spec.adoc:16]: Figured it out.  I can turn off the section numbering for the efsl and turn it back on for the content.

**Comment by kwsutter** [comment]
I just re-used your wombat image.  I have now replaced it with the Jakarta EE boat image.  That probably looks much better and consistent.  What do you think?

**Review by bshannon — CHANGES_REQUESTED** [review]
You must be using a case-insensitive filesystem:

asciidoctor: WARNING: image to embed not found or not readable: /tmp/spec-template/src/main/asciidoc/images/jakartaee.png

**Comment by kwsutter** [comment]
Normally, it's not case insensitive.  It's a Mac...  I must have something set incorrectly for git or maven...  Sorry about that!  I will correct the case and try again...

**Comment by kwsutter** [comment]
Actually, I'm going to make one more tweak...  I'm going to use the official jakarta ee schooner image and use that name in the include statement.

**Review by bshannon — APPROVED** [review]
Look great!  Thanks, Kevin!

**Pull request merged**